README.md

```diff
@@ -1,2 +1,243 @@
+
+### Install
+
+Install this project with:
+Nothing
+No special intallation is required.
+
+### Usage
+
+To run the project:
+Open the directory of the project on your local machine.
+
+### Run tests
+
+To run tests, run the following command:
+Locate and run the index.html file.
+
+### Deployment
+
+You can deploy this project using:
+Any web hosting service. Example: gcloud, aws etc
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author and Collaborators <a name="author-collaborators"></a>
+
+👤 **Omet Omeni (Ometman)**
+
+- GitHub: [@githubhandle](https://github.com/ometman)
+- Twitter: [@twitterhandle](https://twitter.com/ometman)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ometman)
+
+<!-- Mention all of the collaborators of this project.-->
+<!--## 👥 Collaborators <a name="collaborators"></a>
+
+
+👤 **Marcos Hernández**
+
+- GitHub: [@githubhandle](https://github.com/marcoshdezcam)
+
+👤 **Oussama Elabdioui**
+
+- GitHub: [@githubhandle](https://github.com/codedit334)
+
+👤 **Evangelos Fotiadis**
+
+- GitHub: [@githubhandle](https://github.com/vangelif)
+
+👤 **Stephen Katuli**
+
+- GitHub: [@githubhandle](https://github.com/stephenkati) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- 1. **Project Details Modals**
+- 2. **Client-side Form Validation**
+- 3. **In-Browser Data Preservation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, suggestions, issues, and feature requests are welcome!
+
+<!--Feel free to check the [issues page](../../issues/).-->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, do not hesitate to support by contacting me at the social handles above.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all full-time students of Microverse, the morning session teams, learning and coding partners, standup teams and all the student success and support staff. They inspired this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Can I use this project?**
+
+  - Yes, but you can do so with permission from the primary Author.
+
+- **Can I download and modify the code**
+
+  - You are free to download and modify certain sections of the code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE This project is [MIT](./LICENSE) licensed.-->
+
+## 📝 License <a name="license"></a>
+
+This project is MIT licensed. Read the <a href="https://github.com/ometman/My-Portfolio/blob/header-section/README.md">LICENSE</a>.
```

Comment:
- [ ] As requested by the previous code reviewer to avoid the 404 error please remove this part `Read the <a href="https://github.com/ometman/My-Portfolio/blob/header-section/README.md">LICENSE</a>` and add the correct path of the License file 👉🏽 `This project is [MIT](./LICENSE) licensed`. I recommend changing the name of the file `MIT License` by `LICENSE`. It will help you for professional documentation.

Reply:
Okay. Thanks